app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/Date.kt

```diff
@@ -1,8 +1,19 @@
 package com.simplemobiletools.smsmessenger.extensions
 
+import android.content.Context
 import android.text.format.DateFormat
+import android.text.format.DateUtils
+import org.joda.time.DateTime
 import java.util.*
 
 fun Date.format(pattern: String): String {
     return DateFormat.format(pattern, this).toString()
 }
+
+fun DateTime.humanize(context: Context, now: DateTime = DateTime.now(), pattern: String = "EEE, MMM dd, YYYY, hh:mm a"): String {
```

Comment:
lets use the date and time format set in the app settings

Reply:
For the future dates right? That is, we still keep the humanised date format for recent times?

Reply:
lets always use the user selected date and time format, just hide the year if it is the current one

Reply:
[check pls](https://github.com/SimpleMobileTools/Simple-SMS-Messenger/pull/438/commits/f72175478534bc5f7b63fd8e35b4c7490c293e73)